diagnostic.md

```diff
@@ -54,7 +67,11 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
-
+''''CSS
+.big {}
+#contentPane {}
+.important.red {}
+div p {}
```

Comment:
This is a decendent, not a child. the `div > p` will act as the child/parent relationship. Big difference.